### DISCUSSION TITLE
Informal feedback before formal attempt

### DISCUSSION THREAD

**Discussion by Stopwolf** [OpenMined/courses#210] [Project: Privacy Product Specification] [answered]
Is there a way for me to chat with someone about my project idea (to see if I'm going in the right direction, to see if it's possible even 😅) before submitting it and waiting for feedback on the first attempt?

**Comment by yemikifouly** [comment, marked as answer]
Hello! So glad to hear that you have completed the course already! Absolutely! One option will be for you to post your idea here and we can comment on it for you. The advantage is that a wider group of people can give you feedback on it. Alternatively, feel free to DM me about it on Slack. I would be happy to tell you how you are doing. 🙂

**Reply by Stopwolf** [reply]
That's great, thanks!
One idea I had was similar to pol.is. App that aggregates (trending) news (or more generally articles, stories), where experts can validate them after reading, or just post comments of their thoughts. By validating articles as a community of experts, only "true" stories will be pushed to regular readers. This is supposed to incentivise publishing only true stories and make regular readers better informed, better educated, and have them make more educated decisions.

Now the two questions are:
1. how to validate someone as an expert? Maybe via ministry of education of the country if this is a nation-wide product, but this would only go for formal education
2. and also why would experts validate stories/articles on their free time
Another potential problem is if there's a small number of experts in a country, so a regular reader seeing "2 experts validated this story as true" isn't going to have much of an impact on them and their decision making.

Maybe I got stuck to an idea that's too hard or maybe even impossible (currently), but hey.. that's why I'm here asking for your (very appreciated) feedback :smile: 

P.S. This should be mainly for scientific articles, for example articles about covid-19 vaccines, mask efficiencies etc.

**Reply by iamtrask** [reply]
Interesting idea! Another related question is - would individuals trust the experts are neutral? There's been a lot of de-platforming of "experts" these days. I also imagine the people who curate media within communist countries are also called "experts".

Mega important topic - super big challenge. Definitely an interesting project to work on.

**Reply by Stopwolf** [reply]
Thanks @iamtrask !!
Main idea was in a sense more global. So when a big number of experts validate (or disprove) a story, they'll overshadow those biased experts and "experts". On a more global scale the experts should remain neutral. Something similar to the law of big numbers.

But the problem with this being global (and thus also having impact in mentioned communist countries) is exactly validating the experts. It would be very expensive if every country's ministry of education were to validate them.
